go.mod

```diff
@@ -61,13 +35,13 @@ require (
 )
 
 replace (
-	// Result of go get github.com/containerd/containerd@v0.2.x
-	github.com/containerd/containerd => github.com/containerd/containerd v0.2.10-0.20180716142608-408d13de2fbb
-	github.com/docker/docker => github.com/openshift/moby-moby v1.4.2-0.20190308215630-da810a85109d
+	github.com/containerd/containerd => github.com/containerd/containerd v1.3.6
+	github.com/docker/docker => github.com/docker/docker v0.0.0-20190404075923-dbe4a30928d4
```

Comment:
`containerd/containerd` needs to be replaced because there are transitive dependencies which reference different versions that conflict. The 1.3.6 version is a more recent tag that resolves the conflicts.

`docker/docker` is moved to an upstream version with a pseudo-version that replace can resolve. We don't need the OpenShift fork any more as this has drifted from what buildah uses.